convert_util.c

```diff
@@ -109,9 +109,11 @@ convert_parse_tlvs(const uint8_t *buff, size_t buff_len,
 			    CONVERT_ALIGN(sizeof(*conv_connect)))
 				return -1;
 
-			opts->flags		|= CONVERT_F_CONNECT;
-			opts->remote_addr	= conv_connect->remote_addr;
-			opts->remote_port	= conv_connect->remote_port;
+			opts->flags			|= CONVERT_F_CONNECT;
+			opts->remote_addr.sin6_addr	=
+			        conv_connect->remote_addr;
+			opts->remote_addr.sin6_port =
+			        conv_connect->remote_port;
```

Comment:
may you comment that data from conv_connect are already in the right order please?

Reply:
:+1: